spark-plugins/docs/DecisionTreeTrainer-sparksink.md

```diff
@@ -0,0 +1,47 @@
+# DecisionTreeTrainer
+
+
+Description
+-----------
+Trains a regression model based upon a particular label and features of a record. Saves this model to a FileSet.
+
+Use Case
+--------
+This sink can be used when you have a sample data and you want to use it, to build a Decision Tree Regression model.
+This model can be used further for predicting the label.
```

Comment:
This was mentioned to indicate that the model created using the trainer will be used later by the regressor during prediction.
However,  removing this line, to avoid confusion.